app/controllers/reviews_controller.rb

```diff
@@ -14,6 +14,14 @@ def create
     redirect_to books_path(book)
   end
 
+  def destroy
+    # This is very confusing, but our Review ID is passed into params as :book_id because Reviews are nested resources of Book
+    deleted_review = Review.find(params[:book_id])
+    deleted_review.destroy
```

Comment:
As the comment says, this is a big counter-intuitive, but I'm not entirely sure how to avoid it considering our Reviews are nested within Books.